mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/loanaccountdisbursement/LoanAccountDisbursement.java

```diff
@@ -86,6 +81,10 @@
     private DialogFragment mfDatePicker;
     private HashMap<String, Integer> paymentNameIdHashMap = new HashMap<String, Integer>();
 
+    private List<String> paymentType = new ArrayList<>();
+    private ArrayAdapter<String> paymentTypeAdapter;
+    private LoanDisburseTemplate mLoanDisburseTemplate;
```

Comment:
As you have added the LoanDisburseTemplate field, there is no need of 
```private HashMap<String, Integer> paymentNameIdHashMap = new HashMap<String, Integer>();``` field.
Remove this field.